### PR TITLE
New Prototype for new Recorded Future Miner

### PR DESCRIPTION
**Pull request by jtschichold**
#110 Rebased on the most recent code changes

### CODE REVIEW

File: prototypes/recordedfuture.yml [hunk 1]
```diff
@@ -4,11 +4,10 @@ description: >
     enables organizations to defend themselves at the speed and scale of the
     Internet. Our service continuously analyzes content from the entire Web
     to give you unmatched insight into emerging threats.
-    Follow us on Twitter @RecordedFuture
 
 prototypes:
     IPRiskList:
-        author: MineMeld Core Team
+        author: MineMeld Core Team, Recorded Future Inc.
```
Comment (jtschichold) [inline prototypes/recordedfuture.yml]: I would not change the original author of this prototype, but I would mark this as deprecated. The new Master Miner can do more than the original miner.

File: prototypes/recordedfuture.yml [hunk 2]
```diff
@@ -35,7 +31,7 @@ prototypes:
                 share_level: red
 
     DomainRiskList:
-        author: Jonas Eichinger
+        author: Jonas Eichinger, Recorded Future Inc.
```
Comment (jtschichold) [inline prototypes/recordedfuture.yml]: Same here. I would not change the original author of this prototype, but I would mark this as deprecated. The new Master Miner can do more than the original miner.

File: prototypes/recordedfuture.yml [hunk 3]
```diff
@@ -60,3 +53,32 @@ prototypes:
...
+            specified, the default risklist file is used. To access this resource, you need a valid Recorded Future API token.
+        class: minemeld.ft.recordedfuture.MasterRiskList
+        config:
+            confidence: 80
+            age_out:
+                default: null
+                sudden_death: true
+                interval: 3307
+            attributes:
+                share_level: red
```
Comment (jtschichold) [inline prototypes/recordedfuture.yml]: Add a new line at the end of the file

File: prototypes/recordedfuture.yml (hunk 1 above)
Comment (sshah1103) [reply prototypes/recordedfuture.yml]: I will change the name back to its original name. For marking it as deprecated, Should I write something like this :

prototypes:
    IPRiskList:
        author: MineMeld Core Team (deprecated)

We are planning to keep this miner active for our clients.

File: prototypes/recordedfuture.yml (hunk 2 above)
Comment (sshah1103) [reply prototypes/recordedfuture.yml]: I will change the name back to its original name. For marking it as deprecated, Should I write something like this :

prototypes:
IPRiskList:
author: Jonas Eichinger (deprecated)

We are planning to keep this miner active for our clients.

File: prototypes/recordedfuture.yml (hunk 3 above)
Comment (sshah1103) [reply prototypes/recordedfuture.yml]: I will make this change

File: prototypes/recordedfuture.yml (hunk 1 above)
Comment (jtschichold) [reply prototypes/recordedfuture.yml]: you can change `development_status` to `DEPRECATED`

Comment (jtschichold) [reply prototypes/recordedfuture.yml]: and leave the author as it is

File: prototypes/recordedfuture.yml (hunk 2 above)
Comment (jtschichold) [reply prototypes/recordedfuture.yml]: you can change `development_status` to `DEPRECATED` and leave the author as it is

**Comment by jtschichold** [comment]
Closing this in favor of #112

**Pull request closed without merging**